CHANGES.md

```diff
@@ -32,12 +32,15 @@ To be released.
     already received when blockchain is empty.  [[#550], [#562]]
  -  Fixed a bug that `Swarm<T>` had thrown `SocketException` with a message
     `Operation on non-blocking socket would block`.  [[#405], [#485]]
+ -  Fixed a bug that `Swarm<T>` had thrown `InvalidBlockIndexException` during
+    synchronizing another peer that also reorg.  [#528]
```

Comment:
We need to add the link to this PR as well.

Reply:
Fixed.